svc-bip-api/src/main/java/gov/va/vro/bip/model/contentions/GetSpecialIssueTypesResponse.java

```diff
@@ -0,0 +1,38 @@
+package gov.va.vro.bip.model.contentions;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import gov.va.vro.bip.model.BipPayloadResponse;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.jackson.Jacksonized;
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.util.function.Supplier;
+
+@Getter
+@Jacksonized
+@SuperBuilder(toBuilder = true)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class GetSpecialIssueTypesResponse extends BipPayloadResponse
+    implements Supplier<SpecialIssueType[]> {
+
+  // contemplated using ConcurrentInitializer<SpecialIssueType[]>, but have no need for the
+  // concurrency concern given present use case
+  private final Supplier<SpecialIssueType[]> allTypesSupplier;
+
+  public GetSpecialIssueTypesResponse(Supplier<SpecialIssueType[]> allTypesSupplier) {
+    super(BipPayloadResponse.builder());
+    this.allTypesSupplier = allTypesSupplier;
+  }
+
+  public SpecialIssueType[] get() {
+    if (ArrayUtils.isEmpty(codeNamePairs)) codeNamePairs = allTypesSupplier.get();
```

Comment:
For my own awareness, does the team have a convention on single-line if blocks?

Reply:
@brostk, No, no convention on single line ifs.  I am surprised that my IDE didn't fix that, but I am even more surprised that the spotless java style check allowed it. i'll go ahead with the fix so that we don't end up with more of these single lines.